src/Infrastructure/Services/OfficeService/OfficeService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Application.DTOs.OfficeDTO;
+using AutoMapper;
+using Domain.Entities;
+using Infrastructure.Context;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Services.OfficeService
+{
+    public class OfficeService : IOfficeService
+    {
+
+        private readonly ApplicationDbContext _dataContext;
+        private readonly IMapper _mapper;
+
+        public OfficeService(ApplicationDbContext dataContext,IMapper mapper)
+        {
+            _dataContext = dataContext;
+            _mapper = mapper;
+        }
+        public async Task<ServiceResponse<List<GetOfficeDTO>>>  AddOffice(AddOfficeDTO office)
+        {
+            Office office1 = new Office
```

Comment:
dont name like this office1
Change AddOfficeDTO office to AddOfficeDTO officeDto
and then change office1 to newOffice for example